extensions/kubernetes-client/spi/src/main/java/io/quarkus/kubernetes/client/spi/KubernetesDevServiceInfoBuildItem.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.kubernetes.client.spi;
+
+import org.testcontainers.containers.GenericContainer;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class KubernetesDevServiceInfoBuildItem extends SimpleBuildItem {
+    private final String kubeConfig;
```

Comment:
The `kubeConfig` is somehow wider than we actually need and also I think that we already produce `KubernetesClientBuildItem` that should have all the needed configuration. We could possibly remove that, or just keep the token, host and port.

Reply:
To be able to create on the consumer side (example: quarkus-argocd, quarkus-tekton, etc) the Kubernetes client then you need the full string as it will be unmarshalled by the f8 kubernetes client code 
```
        // Create the Kubernetes client using the Kube YAML Config
        KubernetesClient client = new KubernetesClientBuilder()
            .withConfig(io.fabric8.kubernetes.client.Config.fromKubeconfig(kubeServiceInfo.getKubeConfig()))
            .build();
```